src/fprime_gds/common/parsers/seq_file_parser.py

```diff
@@ -99,7 +99,7 @@ def parseArg(arg):
                         # Otherwise it is an enum type:
                         return str(arg)
 
-            return [item for item in map(parseArg, args)]
+            return list(map(parseArg, args))
```

Comment:
Yes, much more concise and in a functional programming sense. The new version is more concise and a bit more efficient by converting the iterable returned by map directly into a list.

Reply:
As for the effectiveness, I'm not so sure. I tried this:

```sh
python3 -m timeit --number=3000000 "
def square(number):
    return number ** 3
numbers = [1, 2, 3, 4, 5]
list(map(square, numbers))
"
3000000 loops, best of 5: 1.68 usec per loop
```

and 

```sh
python3 -m timeit --number=3000000 "
numbers = [1, 2, 3, 4, 5]
[num ** 3 for num in numbers]
"
3000000 loops, best of 5: 1.49 usec per loop
```

But this may be a bias, as the benchmark also includes the definition of the function. Any thoughts?

PS: I still prefer the functional programming style because it is clearer, I just want to understand the finding.

Reply:
@ThibFrgsGmz Thank you for the benchmarking.

I suppose that a fairer comparison in this case would be between this:

`python3 -m timeit --number=3000000 "
def square(number):
    return number ** 3
numbers = [1, 2, 3, 4, 5]
list(map(square, numbers))
"
3000000 loops, best of 5: 1.95 usec per loop`

and

`python3 -m timeit --number=3000000 "
def square(number):
    return number ** 3
numbers = [1, 2, 3, 4, 5]
[item for item in map(square, numbers)]
"
3000000 loops, best of 5: 2.14 usec per loop`

Of course, the performance won't change drastically, but the proposed code is a bit faster and more Pythonic.